app/components/forms/wizard/wizard-footer.hbs

```diff
@@ -33,7 +33,7 @@
   </div>
 </div>
 <div class="ui grid">
-  <div class="sixteen wide column right aligned">
+  <div class="sixteen wide column {{unless this.device.isMobile 'left aligned'}}">
```

Comment:
unless mobile -> right aligned

Reply:
but this will revert the changes. mobile view will be changed to before one as above.

Reply:
I think you are confused between if and unless

Reply:
No I already checked it. But it's showing right aligned despite using unless.